applications/luci-app-tor-hs/htdocs/luci-static/resources/view/tor_hs_common.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+'require view';
+'require dom';
+'require uci';
+'require fs';
+'require ui';
+'require rpc';
+'require form';
+'require tools.widgets as widgets';
+
+function setFlagBool(o) {
+	o.enabled = 'true';
+	o.disabled = 'false';
+}
+
+return view.extend({
+	formdata: { torhs: {} },
+
+	callTorHSList: rpc.declare({
```

Comment:
The common pattern is to define the rpc.declare call not in the view.extend

Reply:
Fixed. Defined outside view.extend